app/src/main/java/org/pesmypetcare/mypetcare/services/user/UserManagerAdapter.java

```diff
@@ -105,8 +104,7 @@ public boolean userExists(User user) {
         ExecutorService executorService = Executors.newSingleThreadExecutor();
         executorService.execute(() -> {
             try {
-                userData.set(ServiceLocator.getInstance().getUserManagerClient().getUser(user.getToken(),
-                        user.getUsername()));
+                userData.set(ServiceLocator.getInstance().getUserManagerClient().getUser(user.getToken(), user.getUsername()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 127).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)